Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM registry.access.redhat.com/ubi8:latest
+USER root
```

Comment:
Maybe this is just my feeble understanding, but do we really need to run the tests as root?

Reply:
Ahhh, never mind...

Reply:
This is just for installation. Tests are run as 1001